quark/mdk-2.0.q

```diff
@@ -788,6 +788,12 @@ namespace mdk {
         }
 
         void start_interaction() {
```

Comment:
It was like that when I got here :( We've discussed plans for having Quark do that, yeah.